src/main/java/io/github/bonigarcia/wdm/cache/CacheHandler.java

```diff
@@ -27,17 +27,15 @@
 import static org.apache.commons.io.FileUtils.listFiles;
 import static org.slf4j.LoggerFactory.getLogger;
 
+import io.github.bonigarcia.wdm.config.Architecture;
```

Comment:
Please avoid changing the order of imports (I use Eclipse for formatting, so this change is not required).